pkg/values/pb/values.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
-// 	protoc        v4.25.1
+// 	protoc-gen-go v1.36.4
+// 	protoc        v5.29.3
```

Comment:
This probably doesn't match the version in CI and will cause failures?

Reply:
Bumped all versions